### PR TITLE
Kamilg/fix bugs

### PR DESCRIPTION
**Pull request by Kamoppl**

### CODE REVIEW

File: test/functional/tests/fault_injection/test_fault_injection_interrupts.py [hunk 1]
```diff
@@ -335,21 +328,16 @@ def test_interrupt_cache_mode_switch_parametrized(cache_mode, stop_percentage):
             core = cache.add_core(core_part)
 
         with TestRun.step(f"Create test file in mount point of exported object."):
-            test_file_size = Size(4, Unit.GibiByte)
+            test_file_size = Size(1024, Unit.MebiByte)
```
Comment (Deixx) [inline test/functional/tests/fault_injection/test_fault_injection_interrupts.py:331]: 1 GiB

File: test/functional/tests/fault_injection/test_fault_injection_interrupts.py [hunk 2]
```diff
@@ -335,21 +328,16 @@ def test_interrupt_cache_mode_switch_parametrized(cache_mode, stop_percentage):
...
-                Dd()
-                .block_size(test_file_size)
-                .input(test_file.full_path)
-                .output(core.path)
-                .oflag("direct")
-            )
-            dd.run()
+            Dd().block_size(test_file_size).input(test_file.full_path).output(core.path).oflag(
+                "direct"
+            ).run()
```
Comment (Deixx) [inline test/functional/tests/fault_injection/test_fault_injection_interrupts.py:340]: didn't you prefer the old version? we talked about it once

File: test/functional/tests/fault_injection/test_fault_injection_interrupts.py [hunk 3]
```diff
@@ -387,7 +375,7 @@ def test_interrupt_cache_mode_switch_parametrized(cache_mode, stop_percentage):
             if cache.get_cache_mode() != cache_mode:
                 TestRun.LOGGER.error("Cache mode should remain the same.")
 
-        with TestRun.step("Stop cache."):
+        with TestRun.step("Unmount core and stop cache."):
```
Comment (Deixx) [inline test/functional/tests/fault_injection/test_fault_injection_interrupts.py:378]: there is no unmount 
sidenote: steps in this test are candidates for review and update

File: test/functional/api/cas/casadm_parser.py [hunk 4]
```diff
@@ -205,11 +233,14 @@ def get_io_class_list(cache_id: int) -> list:
     return ret
 
 
-def get_core_info_by_path(core_disk_path) -> dict | None:
+def get_core_info_by_path(core_disk_path: str, target_cache_id: int) -> dict | None:
```
Comment (Deixx) [inline test/functional/api/cas/casadm_parser.py:236]: needs rename after the update

File: test/functional/api/cas/cache_config.py [hunk 5]
```diff
@@ -85,6 +85,9 @@ def from_name(cls, name):
 
         raise ValueError(f"{name} is not a valid sequential cut off name")
 
+    def __str__(self):
+        return self.value
+
```
Comment (Deixx) [inline test/functional/api/cas/cache_config.py:90]: this is probably not needed after the refactor to string enums

File: test/functional/tests/fault_injection/test_fault_injection_interrupts.py [hunk 6]
```diff
@@ -351,49 +357,56 @@ def test_interrupt_cache_mode_switch_parametrized(cache_mode, stop_percentage):
...
 
-        with TestRun.step("Stop cache."):
+        with TestRun.step("Stop cache"):
             cache.stop()
 
-        with TestRun.step("Check md5 sum of test file again."):
-            Dd().block_size(test_file_size).input(core.path).output(test_file.full_path).oflag(
-                "direct"
-            ).run()
+        with TestRun.step("Compare md5 sum of test files"):
```
Comment (Deixx) [inline test/functional/tests/fault_injection/test_fault_injection_interrupts.py:401]: add a separate step "copy test data from the exported object to a file"

File: test/functional/tests/fault_injection/test_fault_injection_interrupts.py [hunk 7]
```diff
@@ -325,23 +332,22 @@ def test_interrupt_cache_mode_switch_parametrized(cache_mode, stop_percentage):
...
-            test_file_size = Size(4, Unit.GibiByte)
+        with TestRun.step(f"Create test file in mount point of exported object"):
             test_file = fs_utils.create_random_test_file(test_file_path, test_file_size)
 
-        with TestRun.step("Check md5 sum of test file."):
+        with TestRun.step("Calculate md5sum of test file"):
             test_file_md5_before = test_file.md5sum()
 
-        with TestRun.step("Export file to CAS"):
+        with TestRun.step("Copy test file to core"):
```
Comment (Deixx) [inline test/functional/tests/fault_injection/test_fault_injection_interrupts.py:350]: test file -> test data (there will be no file on the core)

File: test/functional/api/cas/cache_config.py (hunk 5 above)
Comment (Kamoppl) [reply test/functional/api/cas/cache_config.py:90]: it is needed

**Review by Deixx — APPROVED** [review]

**Review by katlapinka — APPROVED** [review]

**Pull request merged**